src/docs/product/dashboards/index.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: Dashboards
+sidebar_order: 6
+redirect_from:
+  - /workflow/visibility/
+  - /workflow/dashboards/
+  - /product/error-monitoring/dashboards/
+description: "Sentry's Dashboards help you gain detailed perspectives of your application's health by providing filtering tools to navigate your issues across multiple projects in a single view."
+---
+
+Dashboards help you gain detailed perspectives of your application's health by providing visibility to your errors and transactions across multiple projects in a single view.
+
+Dashboards are comprised of one or more dashboard widgets. Each widget is a visualization of one or more [Discover queries](/product/discover-queries/).
+
+## Overall View of Application
+
+![Widgets visualizing events, errors by country, affected users, and handled vs unhandled issues.](dashboard.png)
+
+When visiting Dashboards for the first time, your organization is provided a set of pre-built widgets. You may edit them to suit your organization's needs.
+
+## Customizing widgets
+
+<Alert title="Note" level="info">
+  Available if your organization is on either the Business plan or the Trial plan.
+</Alert>
+
+![Widget builder.](widget_builder.png)
+
+You can build or edit widgets by:
+
+1. Clicking on "Edit Dashboard" button on the top right hand corner.
+2. Either click on the pencil icon to edit a widget, or clicking on the widget panel with the plus icon.
+
+This will open up the widget builder (see screenshot above).
+
+A widget consists of the following components:
+
+- Widget name
+- Visualization types
+- One or more Discover queries
+  - Search conditions
+  - Fields, tags, or functions
+
+### Visualization types
+
+**Note:** Only functions are available as choices for:
```

Comment:
This is a bit unclear. Do we mean that only one of the following charts is available as a visualization type?

Reply:
This is referring to the available type of choices as outlined in magenta here:

<img width="655" alt="Screen Shot 2021-02-08 at 6 47 35 PM" src="https://user-images.githubusercontent.com/139499/107295816-29fe9980-6a3e-11eb-91ec-77a187e6a5c6.png">


The available type of choices are: fields, tags, measurements, and functions.

I'll add a copy to indicate this.